crates/starknet-devnet-core/src/state/state_diff.rs

```diff
@@ -204,11 +188,12 @@ mod tests {
     }
 
     #[test]
+    #[ignore]
```

Comment:
Something needs to be done about this.

Reply:
The unnecessary write is now removed in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/557/commits/e195917cdc8bbe64a022bebee20f2d7b90a7d751

See https://github.com/0xSpaceShard/starknet-devnet-rs/pull/557#discussion_r1682627653 for more info